webapp/src/components/Modals/SellModal/SellModal.tsx

```diff
@@ -75,8 +79,9 @@ const SellModal = ({
 
   const parsedValueToConfirm = parseFloat(price).toString()
 
-  const isDisabledConfirm =
-    parsedValueToConfirm !== confirmedInput || isCreatingOrder
+  const isDisabledConfirm = isCancel
```

Comment:
What do you think about renaming this variable as such:
```suggestion
  const isConfirmDisabled = isCancel
```
This is because in english, the noun goes before the adjective so it makes the variable easier to read.